unitylibs/core/workflow/workflow-acrobat/target-config.json

```diff
@@ -9,7 +9,9 @@
     "limits": {
       "maxNumFiles": 1,
       "maxFileSize": 104857600,
-      "maxNumPages": 100,
+      "pageLimit": {
+        "maxNumPages": 100
+      },
       "allowedFileTypes": ["application/pdf"],
       "batchSize": 10
```

Comment:
This must've been missed in an earlier PR but let's please remove the `batchSize` property as it is no longer used.